### DISCUSSION TITLE
Not added to contributors list

### DISCUSSION THREAD

**Discussion by dotnetgoo** [CommunityToolkit/Maui#585] [General]
Hi, how are you doing dudes? I'm Mukhammadkarim and founder of .NET Community in Uzbekistan. 

.NET MAUI is very interesting for me. I'm going to contribute your projects. Also, I've created some changes and PR. Reviewers has approved it. I have not yet joined the ranks of contributors. I'm sorry if I said something wrong. You can see in following image:

<img width="494" alt="image" src="https://user-images.githubusercontent.com/75162831/188231046-9368e949-a7c8-4bf4-8a59-5b0603e1df63.png">

**Comment by VladislavAntonyuk** [comment]
I think it's updated once per day

**Reply by dotnetgoo** [reply]
Cool. Thanks bro )

**Reply by VladislavAntonyuk** [reply]
Thank you for your contribution!

**Reply by pictos** [reply]
@MrTukhtaboev I can see you in the contributiors list 

![image](https://user-images.githubusercontent.com/20712372/188247522-db328576-af05-4064-8e5f-cd51eb28d85a.png)

**Reply by dotnetgoo** [reply]
@pictos Thanks a lot ) I hava seen

**Comment by CliffAgius** [comment]
Closed as answered
